Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/Undo/PrefabUndoUtils.h

```diff
@@ -27,18 +26,26 @@ namespace AzToolsFramework
                 const PrefabDomValue& newEntityDom,
                 const AZStd::string& newEntityAliasPath);
 
-            //! Create remove-entity patch from entity's alias path, and append it to a given instance 
-            //! patch which is an array of entity patches.
+            //! Create remove patch from an alias path, and append it to a given instance patch.
             //! @param patch An array object of DOM values which stores entity patches.
```

Comment:
Should we remove the word "entity" from this comment?

Reply:
yeah! I am reviewing all the comments in this util class to make sure they are correct and accurate.